# Solaris-specific system paths.
# Our bash configuration lives in /etc/bash.
# Not for upstream.
--- lib/readline/rlconf.h	2009-01-04 11:32:33.000000000 -0800
+++ lib/readline/rlconf.h	2010-09-01 10:48:13.018894232 -0700
@@ -39,7 +39,7 @@
 #define DEFAULT_INPUTRC "~/.inputrc"
 
 /* The ultimate last-ditch filenname for an init file -- system-wide. */
-#define SYS_INPUTRC "/etc/inputrc"
+#define SYS_INPUTRC "/etc/bash/inputrc"
 
 /* If defined, expand tabs to spaces. */
 #define DISPLAY_TABS
